README.md

```diff
@@ -34,24 +168,17 @@ scrape_configs:
 
   # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
   - job_name: 'maia'
-    metrics_path: "/api/v1/metrics"
+    metrics_path: "/federate"
```

Comment:
Any reason to break the `/api/v1` pattern? This could get confusing. E.g. if the underlying implementation introduces breaking changes and we want to introduce a 2nd version.